seqr/views/utils/dataset_utils.py

```diff
@@ -158,7 +158,6 @@ def _find_or_create_missing_sample_records(
                 **kwargs
             ) for sample_key, individual in sample_id_to_individual_record.items()]
         samples += list(Sample.bulk_create(user, new_samples))
-        log_model_bulk_update(logger, new_samples, user, 'create')
```

Comment:
`Sample.bulk_create`  actually calls `log_model_bulk_update` so we had a duplicate of this log